src/main/java/org/folio/dew/batch/bursarfeesfines/AccountReader.java

```diff
@@ -1,22 +1,29 @@
 package org.folio.dew.batch.bursarfeesfines;
 
+import com.google.inject.Key;
```

Comment:
Are all these still necessary? @nhanaa 

Clean up the imports then you should be good to merge